orix/quaternion/quaternion.py

```diff
@@ -498,3 +499,72 @@ def _outer_dask(
 
         new_chunks = tuple(chunks1[:-1]) + tuple(chunks2[:-1]) + (-1,)
         return out.rechunk(new_chunks)
+
+    @classmethod
+    def from_scipy_rotation(cls, rotation: SciPyRotation) -> Quaternion:
+        """Return quaternion(s) from :class:`scipy.spatial.transform.Rotation`(s).
```

Comment:
The final "(s)" apparently makes the Sphinx class directive not work, apparently.

```suggestion
        """Return quaternion(s) from :class:`scipy.spatial.transform.Rotation`.
```